Config.cmake/runtime_test.cmake

```diff
@@ -61,24 +61,20 @@ target_link_libraries(jm_vector_test PRIVATE jmutils)
 target_include_directories(jm_vector_test PRIVATE ${FMIL_TEST_INCLUDE_DIRS})
 
 # Test: jm locale
-add_executable(jm_locale_test ${FMIL_TEST_DIR}/jm_locale_test.c)
-target_link_libraries(jm_locale_test jmutils)
-target_include_directories(jm_locale_test PRIVATE ${FMIL_TEST_INCLUDE_DIRS})
 if(FMILIB_TEST_LOCALE)
-    target_compile_definitions(jm_locale_test PRIVATE -DFMILIB_TEST_LOCALE ${FMIL_LINK_WITH_SUBLIBS})
+    add_executable(jm_locale_test ${FMIL_TEST_DIR}/jm_locale_test.c)
+    target_link_libraries(jm_locale_test jmutils)
+    target_include_directories(jm_locale_test PRIVATE ${FMIL_TEST_INCLUDE_DIRS})
+    target_compile_definitions(jm_locale_test PRIVATE ${FMIL_LINK_WITH_SUBLIBS})
```

Comment:
This one seems to be redundant with the test change?

Reply:
I don't think so. We still make use of `jm_get_default_callbacks` which is defined in the header as:

```
FMILIB_EXPORT
jm_callbacks* jm_get_default_callbacks(void);
```

And FMILIB_EXPORT will expand to `__declspec(dllimport)` unless we set that flag, which will cause the issue with it expecting to link with an import lib instead of a static lib (see `config_fmilib.h.cmake` for details on how FMILIB_EXPORT is expanded).

Reply:
Ah ok, thanks for the clarification, I somehow missed that link.